vllm/_custom_ops.py

```diff
@@ -1507,7 +1507,7 @@ def scaled_fp8_quant(
                 output, input, scale, scale_ub
             )
         else:
-            scale = torch.empty(1, device=input.device, dtype=torch.float32)
+            scale = torch.empty((1, 1), device=input.device, dtype=torch.float32)
```

Comment:
Why is this needed? AFAIK this tensor is just a scalar to the kernel

Reply:
The native implementation returns (1,1) so this just makes them consistent. I don't remember exactly what I was running into